jetstream/pre_treatment.py

```diff
@@ -60,6 +60,39 @@ def apply(self, df: DataFrame, col: str) -> DataFrame:
         return df.loc[mask, :]
 
 
+@attr.s(auto_attribs=True)
+class CensorLowestValues(PreTreatment):
+    """Removes rows with the lowest n% of values."""
+
+    fraction: float = 1e-5
+
+    def apply(self, df: DataFrame, col: str) -> DataFrame:
+        mask = df[col] > df[col].quantile(self.fraction)
+        return df.loc[mask, :]
+
+
+@attr.s(auto_attribs=True)
+class CensorValuesBelowThreshold(PreTreatment):
+    """Removes rows with values below the provided threshold."""
+
+    threshold: float = 0.5
```

Comment:
Can you remove the default value for this and CensorValuesAboveThreshold? I don't think it makes sense to use these without choosing a value intentionally.